plugins/JsTrackerInstallCheck/plugin.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "JsTrackerInstallCheck",
```

Comment:
Plugins that are directly integrated in core don't need a `plugin.json`. That way they will always have the same version as core.

Reply:
I didn't originally have one, but then there was no description in the list of activated plugins. If that's alright or there's a different way to set that description, I'm happy to remove this file. @bx80 Do you know of a way to set the plugin description outside of the plugin.json file?

Reply:
@snake14 you can simply provide `PluginDescription` in the language file of the plugin.